sysdeps/unix/sysv/linux/clone-internal.c

```diff
@@ -106,7 +106,7 @@ __clone_internal (struct clone_args *cl_args,
   __set_errno (saved_errno);
 #endif
 
-  return __clone_internal_fallback (cl_args, func, arg);
+  return 0;
```

Comment:
why we change the return value here?

Reply:
updated the comment. Since this function would crash for some reason, and this branch should also not be reached normally.